### PR TITLE
Make sure footnote displays even with continue button

### PR DESCRIPTION
**Pull request by drewsamnick**
This change makes sure that the footnote doesn't get hidden when we hide the signup button. I also moved the continue button to be below the footnote (same place that the signup button appears.)

Before:
screen shot 2017-10-12 at 1 33 22 pm

After:
screen shot 2017-10-12 at 1 34 23 pm

### CODE REVIEW

**Comment by drewsamnick** [comment]
[Move the text under the submit button on the second page of HOC questions to the first page above the new "submit and continue button"](https://trello.com/c/df32VOt6/21-move-the-text-under-the-submit-button-on-the-second-page-of-hoc-questions-to-the-first-page-above-the-new-submit-and-continue-bu)

File: pegasus/sites.v3/hourofcode.com/views/signup_form.haml [hunk 1]
```diff
@@ -106,20 +106,20 @@
...
-          = hoc_s(:signup_submit_and_continue_label)
-
     #submit
       %input{name:'hoc_country_s', type:'hidden', value:@country}
       %input{name:'hoc_company_s', type:'hidden', value:@company}
       %p.footnote
         = hoc_s(:signup_multiple_event_warning)
         %strong= hoc_s(:signup_registration_deadline)
-      .form-group.submit-btn
+      .form-group.submit-btn#submit-button
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/views/signup_form.haml]: I'm confused by the inconsistency between the two submit buttons. Should the id and class be applied to the `.form-group` or the `%button` itself?

File: pegasus/sites.v3/hourofcode.com/public/js/signup-form.js [hunk 2]
```diff
@@ -44,23 +44,23 @@ $(document).ready(function () {
...
+      $('#submit-button').show();
     } else if ($('#hoc-event-type').val() === 'out_of_school') {
       // out of school, either US or non-US
       $('#organization-name-field').show();
       $('#hoc-event-location-field').show();
       $('#school-name-field').hide();
       $('#hoc-entire-school').hide();
       $('.continue-btn').hide();
-      $('#submit').show();
+      $('#submit-button').show();
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/public/js/signup-form.js]: I would expect to see #continue-btn and #submit-btn or #continue-button and #submit-button but not .continue-btn and #submit-button. Can we make them more consistent?

Comment (drewsamnick) [reply pegasus/sites.v3/hourofcode.com/public/js/signup-form.js]: Done.

File: pegasus/sites.v3/hourofcode.com/public/js/signup-form.js [hunk 3]
```diff
@@ -101,7 +101,7 @@ $(document).ready(function () {
 function showCensusForm(data) {
   $('.main-form').hide();
   $('#signup-header').hide();
-  $('.continue-btn').hide();
+  $('#submit').hide();
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/public/js/signup-form.js:104]: Was this change meant to be `#continue-btn` instead of `#submit`?

**Pull request merged**